### PR TITLE
feat: add github actions for opa rego testing

### PR DESCRIPTION
**Pull request by oycyc**
GitHub Actions that automates testing OPA policies on GitHub and also generates a report including coverage and test results!

### CODE REVIEW

File: README.md
Comment (anderseknert) [inline README.md:303]: Can we use the title of the project for the text here instead? I.e:

```
[github-action-opa-rego-test](https://github.com/masterpointio/github-action-opa-rego-test) - GitHub Action to automate testing for your OPA Rego policies and generate a report.
```

It feels like the text is mostly repeated in the description later anyway.

**Review by anderseknert — COMMENTED** [review]
Thanks for contributing to the OPA ecosystem! Looks like a useful project 😃 Just a question/nit to address before we merge this.

Good work!

File: README.md
Comment (oycyc) [reply README.md]: I also made a PR over here https://github.com/open-policy-agent/opa/pull/6954
I believe you're a maintainer there as well? Would be nice to get it added there too! Thanks!

**Review by anderseknert — APPROVED** [review]

**Comment by anderseknert** [comment]
Thanks! 😃

**Pull request merged**
